mat64/gsvd.go

```diff
@@ -0,0 +1,296 @@
+// Copyright ©2017 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+// Based on the SingularValueDecomposition class from Jama 1.0.3.
+
+package mat64
+
+import (
+	"github.com/gonum/blas/blas64"
+	"github.com/gonum/floats"
+	"github.com/gonum/lapack"
+	"github.com/gonum/lapack/lapack64"
+	"github.com/gonum/matrix"
+)
+
+// GSVD is a type for creating and using the Generalized Singular Value Decomposition
```

Comment:
I want to say "Surely there's a description somewhere of why one wants this". I did some quick looking and failed to find anything more useful. But it still remains that presumably you're implementing this for a purpose. Maybe a link to that is more useful than nothing?